### PR TITLE
Update how-to-create-azure-support-request.md

### PR DESCRIPTION
**Pull request by thiruma0203**
Added behavior of "Save contact changes for future support requests" checkbox and "Additional email for notification", that has been confirmed as "by design" by product team.

### CODE REVIEW

**Comment by Jak-MS** [comment]
@JnHs 
Can you review the proposed changes? 

**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: articles/azure-portal/supportability/how-to-create-azure-support-request.md [hunk 1]
```diff
@@ -101,6 +101,9 @@ Next, we collect more details about the problem. Providing thorough and detailed
 
 1. Complete the **Contact info** section so that we know how to reach you.
 
+  > [!TIP]
+  > By enabling "Save contact changes for future support requests", your personal contact info is saved for future use. However, please note that "Additional email for notification" is not saved.
```
Comment (JnHs) [inline articles/azure-portal/supportability/how-to-create-azure-support-request.md:105]:
```suggestion
  > By enabling **Save contact changes for future support requests**, your personal contact info is saved for future use. However, the **Additional email for notification** value is not saved.
```

**Review by JnHs — CHANGES_REQUESTED** [review]
Small corrections for style consistency

**Comment by JnHs** [comment]
@MicrosoftDocs/public-repo-pr-review-team are you able to make the requested change & merge this PR? If so, please do so at your convenience - or does the original author need to do that?

**Comment by Court72** [comment]
Hi @JnHs,

Thanks for reviewing and tagging us! The PR review team can commit suggested changes for style, spelling, formatting, etc. We leave substantial revisions to the original contributor and article author.
Merging now.

**Pull request merged**